PocketKit/Sources/PocketKit/Tags/TaggedFilter/TagsFilterToolbar.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Textile
+import Localization
+import Sync
+
+struct TagsFilterToolBar: ViewModifier {
+    @Binding var editMode: EditMode
+    @ObservedObject var viewModel: TagsFilterViewModel
+
+    func body(content: Content) -> some View {
+        content
+            .toolbar {
+                ToolbarItem(placement: .principal) {
+                    Text(Localization.tags).style(.tags.sectionHeader)
+                }
+                ToolbarItem(placement: .navigationBarLeading) {
+                    Image(asset: .tag).foregroundColor(Color(.ui.grey5))
```

Comment:
Should all these have accessibilityIdentifiers?

Reply:
we can definitely add them. My guess is that they weren't added because the other two aren't actionable items, but certainly does not hurt to have them